app/models/navigation_link.rb

```diff
@@ -4,12 +4,16 @@ class NavigationLink < ApplicationRecord
   before_validation :allow_relative_url, if: :url?
   before_save :strip_local_hostname, if: :url?
 
+  enum section: { default: 0, other: 1 }, _suffix: :section
```

Comment:
You can simplify this a bit:

```suggestion
  enum section: { default: 0, other: 1 }, _suffix: true
```

To quote [the docs](https://api.rubyonrails.org/v5.2.3/classes/ActiveRecord/Enum.html):

> You can use the` :_prefix` or `:_suffix `options when you need to define multiple enums with same values. If the passed value is `true`, the methods are prefixed/suffixed with the name of the enum.

Reply:
TIL!